graylog2-web-interface/src/components/graylog/Alert.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { forwardRef, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+// eslint-disable-next-line no-restricted-imports
+import { Alert as BootstrapAlert } from 'react-bootstrap';
+
+import { useTheme } from 'theme/GraylogThemeContext';
+import bsStyleVariant from './variants/bsStyle';
+
+const Alert = forwardRef(({ bsStyle, ...props }, ref) => {
```

Comment:
Just out of interest, I understand what `forwardRed` does, but why do we need it in this case?

Reply:
Because the ref doesn't get forwarded to the actual button if we don't, so if you try and use `<Button ref={...} />`it will give you React errors

Reply:
alright!